### PR TITLE
RGW|BN: make http tests run on a seperate task

### PR DESCRIPTION
**Pull request by AliMasarweh**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Comment by AliMasarweh** [comment]
teuthology run
https://pulpito.ceph.com/alimasa-2024-06-05_10:16:30-rgw:notifications-wip-alimasa-bn-end-point-distro-default-smithi/

**Comment by yuvalif** [comment]
you should also remove `http_test` fro mthe task that ran them so far

File: qa/suites/rgw/notifications/tasks/http/supported-distros/ubuntu_latest.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+os_type: ubuntu
+os_version: "22.04"
+ktype: distro
```
Comment (cbodley) [inline qa/suites/rgw/notifications/tasks/http/supported-distros/ubuntu_latest.yaml:3]: these distro files should always be symlinks, please don't create copies

`tasks/http/supported-distros` should be a symlink to `.qa/distros/supported-random-distro$`

File: qa/suites/rgw/notifications/tasks/http/supported-distros/$ [hunk 2]
```diff
@@ -0,0 +1 @@
+qa/suites/rgw/notifications/tasks/kafka/supported-distros/$
```
Comment (yuvalif) [inline qa/suites/rgw/notifications/tasks/http/supported-distros/$:1]: i think you shoudl link here: `../../.qa/distros/supported-random-distro$/`
not to kafka

**Review by yuvalif — APPROVED** [review]

**Comment by AliMasarweh** [comment]
https://pulpito.ceph.com/alimasa-2024-06-06_11:37:40-rgw:notifications-wip-alimasa-bn-end-point-distro-default-smithi/

**Pull request merged**